monai/bundle/scripts.py

```diff
@@ -608,16 +608,19 @@ def run(
     .. code-block:: bash
 
         # Execute this module as a CLI entry:
-        python -m monai.bundle run training --meta_file <meta path> --config_file <config path>
+        python -m monai.bundle run --meta_file <meta path> --config_file <config path>
+
+        # Execute with specified `run_id=runtest`, `init_id=inittest`, `final_id=finaltest`:
+        python -m monai.bundle run --run_id runtest --init_id inittest --final_id finaltest ...
```

Comment:
Hi @ericspod ,

Please check this newly added example, I added this more clear example for `run_id`, `init_id` and `final_id`.

Thanks.

Reply:
That's good but it can be done without `--run_id` correct? Our old command line examples still work so one instance here would help. I think users find it hard to understand that the command line here is representing the arguments for the function call.

Reply:
OK, I added 1 more old example also.

Thanks.